components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -213,41 +221,49 @@ const TrialBenefitsModal = ({
         svgWrapperClassName,
         svgElement,
         bottomLeftMessage,
+        isCloud,
     }: TrialBenefitsModalStepProps) => {
+        // when we are in cloud, ommit the cta to go to the system console, this because the license changes take some time to get applied (see MM-44463)
+        // also, the design of the modal changes a little bit by placing the svg image on top and the title changes too.
```

Comment:
Same here as above, any design changes should be applied to both and not make conditionals for just cloud